### PR TITLE
Clarify headers on security tutorial

### PR DESCRIPTION
**Pull request by cjohnston1158**
## Issue
#267 

## Solution
Updates headers to clarify secions.

### CODE REVIEW

**Review by dragomirp — APPROVED** [review]

**Comment by dragomirp** [comment]
We'll need to update discourse as well.

**Comment by cjohnston1158** [comment]
> We'll need to update discourse as well.

[Done](https://discourse.charmhub.io/t/charmed-postgresql-k8s-tutorial-enable-security/9302).

**Review by marceloneppel — APPROVED** [review]

**Comment by taurus-forever** [comment]
Thank you for taking care, reporting and fixing it!
I have applied the same changes for PostgreSQL [VM](https://discourse.charmhub.io/t/charmed-postgresql-tutorial-enable-security/9699), MySQL [VM](https://charmhub.io/mysql/docs/t-enable-security)/[K8s](https://discourse.charmhub.io/t/charmed-mysql-k8s-tutorial-enable-security/9669).

I am merging this PR to close the issue. Hovewer, we are in process of automated Documentation [sync](https://github.com/canonical/mysql-k8s-operator/pull/306) from Discourse to GH.
TL;DR: just edit Wiki on Discouse and PR to GitHub will be created automatically. Tnx!
